### DISCUSSION TITLE
Replaced deleted / removed titles

### DISCUSSION THREAD

**Discussion by moserroger** [croneter/PlexKodiConnect#1898] [General]
Hello everyone

Unfortunately I didnt find an answer to the following situation so far. I am sorry if I overlooked a source.

I have the following situation:
Kodi 17 installed, PlexKodiConnect which grabs the media on an updated Plex server. Recently I had to change some files on the server. At the same time I wanted to watch a series that was also affected by the changes. The data on Kodi wasn't yet updated and the program told me that the title wasn't available and asked me if I wanted to delete the episode. Which I unfotunately did. Now the rest of the episodes work again and even if I update the Kodi library the missing title will not reappear. How update or how to procede to retrieve or add the missing title again? The data on Plex is there.

Thank you for your help.

moses

**Comment by croneter** [comment]
Reset the PlexKodiConnect database in the settings under Advanced. Do NOT reset the PKC add-on when it asks you.

The issue here is that Plex unfortunately does not tell when it replaced a file name. There is simply no way for PKC to know that a file name changed on the Plex side 😢. It works if anything else changes, like a movie's description

**Comment by moserroger** [comment]
Thank you. It worked.

    On Friday, December 30, 2022 at 07:20:27 AM GMT+1, croneter ***@***.***> wrote:  
 
 


Reset the PlexKodiConnect database in the settings under Advanced. Do NOT reset the PKC add-on when it asks you.

The issue here is that Plex unfortunately does not tell when it replaced a file name. There is simply no way for PKC to know that a file name changed on the Plex side 😢 . It works if anything else changes, like a movie's description
